arbnode/sequencer.go

```diff
@@ -46,7 +46,7 @@ type Sequencer struct {
 	txStreamer      *TransactionStreamer
 	txQueue         chan txQueueItem
 	l1Reader        *headerreader.HeaderReader
-	config          SequencerConfig
+	configFetcher   SequencerConfigFetcher
```

Comment:
Maybe we could just call this `config` instead of `configFetcher` to save a few characters on a commonly accessed field